osf_tests/test_user.py

```diff
@@ -1272,17 +1275,6 @@ def test_bookmark_collection_nodes_arent_merged(self, dupe, master, merge_dupe):
         merge_dupe()
         assert not master.collection_set.filter(id=dashnode.id).exists()
 
-    # Note the files are merged, but the actual node stays with the dupe user
-    def test_quickfiles_node_arent_merged(self, dupe, master, merge_dupe):
-        assert master.nodes.filter(type='osf.quickfilesnode').count() == 1
-        assert dupe.nodes.filter(type='osf.quickfilesnode').count() == 1
-
-        merge_dupe()
-        master.refresh_from_db()
-        dupe.refresh_from_db()
-        assert master.nodes.filter(type='osf.quickfilesnode').count() == 1
-        assert dupe.nodes.filter(type='osf.quickfilesnode').count() == 1
```

Comment:
This test ensures a quickfilesnode stays with the merged user, since there's no target quickfilesnode after PR this the target node won't exsist and this test will be failing/irrelevant.

Reply:
👍  Moving those tests from `quickfiles/test_models.py` here.